livre/eq/Window.cpp

```diff
@@ -69,10 +69,12 @@ struct Window::Impl
                         pipe->getFrameData()->getVRParameters().getMaxGPUCacheMemoryMB();
         _textureCache.reset( new TextureCache( node->getTextureDataCache(),
                                                maxGpuMemory * LB_1MB, GL_LUMINANCE8 ));
-
-        const size_t computeThreads = 4;
+        const size_t renderThreads = 2;
+        const size_t computeThreads = 2;
         const size_t uploadThreads = 4;
```

Comment:
I'd move these constants to RenderPipeline and remove the arguments. Add the arguments if you're really going to change the values, if not they are just making the API thicker for no good reason.